src/main/java/be/seeseemelk/mockbukkit/block/state/BlastFurnaceMock.java

```diff
@@ -26,7 +26,6 @@ protected BlastFurnaceMock(@NotNull Block block)
 	@Override
 	protected @NotNull InventoryMock createInventory()
 	{
-//		return new BlastFurnaceInventoryMock(this); TODO: Not implemented
 		return new InventoryMock(this, InventoryType.BLAST_FURNACE);
```

Comment:
This needs to return the new inventory.
```suggestion
		return new FurnaceInventoryMock(this);
```